### PR TITLE
Updates the root readme file for 2023

### PR DESCRIPTION
**Pull request by michaelnoonan**
I realised this didn't point to our handbook and it had some older concepts. This change brings it into line with 2023.

### CODE REVIEW

**Review by mattocto — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -44,20 +35,14 @@ We have identified several **defining characteristics** about each role at Octop
...
-  - It makes sense to record the coaching plan.
-
-Let's call special attention to clearly outstanding/exemplary behaviour:
-
-- `Outstanding`: _You simply shine in this area like very few others. I think people deserve to know this about you._
-  - This is more than just saying "I'm really, really good at this". It's saying that you're right up there with the best few across all of Octopus at this.
+- `Performing`: _"You're great at this! You're doing exactly what we expect of someone in this role and level. Keep up the great work!"_
+- `Exceeding`: _"In this area, you are exceeding what we expect of someone in this role and level."_
+- `Maturing`: _"You've got some growth required in this area, but the trajectory is good. Here's some suggestions, take them or leave them - it's your call."_
+- `Coaching needed`: _"You've got some growth required in this area and the trajectory isn't good. Let's work out a coaching plan and turn this around together."_
```
Comment (MarkSiedle) [inline README.md:45]: I've typically seen us associate the words "coaching" and "supporting" _within_ the maturing role (from the Situation Leadership model), and words such as "improvement needed" to indicate that there's a clear problem/gap that needs addressing and the trajectory is not good. I.e. "Improvement needed" is effectively the signal before more drastic measures or harder conversations need to be considered (early signal towards either performance management 😞  or improvement taken 😌 ).

Is there a risk that "coaching" becomes associated with "my trajectory is not good"?

If so, an alternative that aligns with the HiBob/SI ratings we've given in the past would be "Improvement needed". And perhaps some changes to the maturing above to clarify a spectrum:

E.g.
> ... Here's some suggestions, take them or leave them - it's your call; Here's some coaching we'll work through together to help ensure you are setup to succeed on your maturing journey."_

**Review by MarkSiedle — APPROVED** [review]
One thing for us to consider, but this is a big improvement (much less text and more clarity :chefkiss:)

File: README.md (hunk 1 above)
Comment (MarkSiedle) [reply README.md:45]: Tough one. With the SL model, I can also see how coaching implies they are in that "warning" state, so perhaps it is a good signal-term that the trajectory is not (yet) good 👍

Comment (michaelnoonan) [reply README.md:45]: Good call @MarkSiedle . Can you please make a suggestion to this file?

**Review by MarkSiedle — COMMENTED** [review]
Suggestions inline

**Pull request merged**